<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructureWindow</class>
 <widget class="QMainWindow" name="StructureWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Structure window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="StructureView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Tool bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_room"/>
   <addaction name="actionEditRoom"/>
   <addaction name="actiondeleteRoom"/>
  </widget>
  <widget class="QDockWidget" name="roomInfoDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Room details</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="roomInfoDockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="roomInfoDockLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="3" column="2">
        <widget class="QLabel" name="heightLabel">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="roomNameLineEdit"/>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="roomNameLabel">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="buddy">
          <cstring>roomNameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="roomNumberLabel">
         <property name="text">
          <string>Position (nÂ°)</string>
         </property>
         <property name="buddy">
          <cstring>roomNumberSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="widthLabel">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="roomNumberSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" rowspan="2">
        <widget class="Line" name="sizeSepLine">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QLabel" name="sizeLabel">
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="roomWidthSpinBox">
         <property name="suffix">
          <string> m</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="roomHeightSpinBox">
         <property name="suffix">
          <string> m</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew_room">
   <property name="icon">
    <iconset resource="../../images.qrc">
     <normaloff>:/icons/icons/NewFile.png</normaloff>:/icons/icons/NewFile.png</iconset>
   </property>
   <property name="text">
    <string>New room</string>
   </property>
  </action>
  <action name="actiondeleteRoom">
   <property name="text">
    <string>Delete room</string>
   </property>
  </action>
  <action name="actionEditRoom">
   <property name="text">
    <string>Edit room</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StructureView</class>
   <extends>QGraphicsView</extends>
   <header location="global">GUI/Widgets/structureview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
